ecosystem/python/sdk/aptos_sdk/async_client.py

```diff
@@ -742,6 +742,27 @@ async def get_collection(
             "0x3::token::CollectionData",
             collection_name,
         )
+    
+    async def transfer_object(
+        self, owner: Account, object: AccountAddress, to: AccountAddress
+    ) -> str:
+        transaction_arguments = [
+            TransactionArgument(object, Serializer.struct),
+            TransactionArgument(to, Serializer.struct),
+        ]
+
+        payload = EntryFunction.natural(
+            "0x1::object",
+            "transfer_call",
+            [],
+            transaction_arguments,
+        )
```

Comment:
We may want to make this the object input one, so we give people good fundamentals.